Code/GraphMol/MolDraw2D/DrawText.cpp

```diff
@@ -64,14 +64,18 @@ void DrawText::setMinFontSize(double new_min) { min_font_size_ = new_min; }
 double DrawText::fontScale() const { return font_scale_; }
 
 // ****************************************************************************
-void DrawText::setFontScale(double new_scale) {
+void DrawText::setFontScale(double new_scale, bool ignore_extremes) {
   font_scale_ = new_scale;
+  if (ignore_extremes) {
+    return;
+  }
+
```

Comment:
```suggestion
void DrawText::setFontScale(double new_scale, bool ignoreExtremes) {
  font_scale_ = new_scale;
  if (ignoreExtremes) {
    return;
  }

```

Reply:
minor change to try and avoid having the API get even more inconsistent in terms of naming